modules/gateway/gateway_test.go

```diff
@@ -0,0 +1,76 @@
+package gateway
+
+import (
+	"strconv"
+	"testing"
+
+	"github.com/NebulousLabs/Sia/consensus"
+	"github.com/NebulousLabs/Sia/modules"
+)
+
+var port = 9001
```

Comment:
well, you could throw a lock on the port. I actually tried that when I was trying to make the GatewayTester work, and it seemed to help a bit, but it didn't fix the core problem.